endpoints/bahmni/input-validation.json

```diff
@@ -1,553 +1,203 @@
                 "type": "string"
+              },
+              "countyDistrict": {
+                "type": "string"
+              },
+              "startDate": {
+                "type": "string",
+                "format": "date-time"
+              },
+              "endDate": {
+                "type": "string"
+              },
+              "address1": {
+                "type": "string",
+                "nullable": true
+              },
+              "address2": {
+                "type": "string",
+                "nullable": true
+              },
+              "address3": {
+                "type": "string",
+                "nullable": true
+              },
+              "address4": {
+                "type": "string",
+                "nullable": true
+              },
+              "address5": {
+                "type": "string",
+                "nullable": true
+              },
+              "address6": {
+                "type": "string",
+                "nullable": true
+              },
+              "address7": {
+                "type": "string",
+                "nullable": true
+              },
+              "address8": {
+                "type": "string",
+                "nullable": true
+              },
+              "address9": {
+                "type": "string",
+                "nullable": true
+              },
+              "address10": {
+                "type": "string",
+                "nullable": true
+              },
+              "address11": {
+                "type": "string",
+                "nullable": true
+              },
+              "address12": {
+                "type": "string",
+                "nullable": true
+              },
+              "address13": {
+                "type": "string",
+                "nullable": true
+              },
+              "address14": {
+                "type": "string",
+                "nullable": true
+              },
+              "address15": {
+                "type": "string",
+                "nullable": true
               }
             }
           }
-        }
-      },
-      "dateCreated": {
-        "type": "timestamp"
-      },
-      "changedBy": {
-        "type": "string"
-      },
-      "dateChanged": {
-        "type": "timestamp"
-      }
-    },
-    "deathdateEstimated": {
-      "type": "boolean",
-      "default": false
-    },
-    "birthtime": {
-      "type": "timestamp"
-    },
-    "links": {
-      "type": "array",
-      "element": {
-        "rel": {
-          "type": "string"
-        },
-        "uri": {
-          "type": "string"
-        }
-      }
-    },
-    "resourceVersion": {
-      "type": "string"
-    }
-  },
-  "voided": {
-    "type": "boolean"
-  },
-  "auditInfo": {
-    "type": "object",
-    "element": {
-      "creator": {
-        "uuid": {
-          "type": "string"
         },
-        "display": {
+        "gender": {
           "type": "string"
```

Comment:
@bradsawadye This could use more validation:

```suggestion
          "type": "string",
          "enum": ["F", "M", "O", "U"]
```

Reply:
fixed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/da90c901b05f8fcd2480d722da8ccaa7f6a86d77